tests/TestCase.php

```diff
@@ -62,6 +63,10 @@ protected function tearDown(): void
     {
         Str::createUuidsNormally();
 
+        if (method_exists(WorkCommand::class, 'flushState')) {
+            WorkCommand::flushState();
+        }
```

Comment:
Fixes a test issue in some versions where event listeners don't get re-applied due to the static property surviving between tests.